examples/api/lib/widgets/animated_switcher/animated_switcher.0.dart

```diff
@@ -37,7 +38,7 @@ class _AnimatedSwitcherExampleState extends State<AnimatedSwitcherExample> {
         mainAxisAlignment: MainAxisAlignment.center,
         children: <Widget>[
           AnimatedSwitcher(
-            duration: const Duration(milliseconds: 500),
+            duration: const Duration(microseconds: 500),
```

Comment:
```suggestion
            duration: const Duration(milliseconds: 500),
```

microseconds is too fast.

Reply:
@TahaTesser oh, I didn't notice this, thanks!